modules/tensor_mechanics/doc/content/source/utils/RankTwoScalarTools.md

```diff
@@ -40,24 +40,13 @@ where $D$ is the direction vector specified in the input file.
 
 ## Effective Strain
```

Comment:
should this heading be effective strain Increment

Reply:
Well, ultimately what we want to compute is the effective strain. The increment is just the means here. But you're right, in this tools namespace we only deal with the increment. The integration is performed in the material.